src/flow-typed/npm/moment_v2.3.x.js

```diff
@@ -0,0 +1,331 @@
+// flow-typed signature: 23b805356f90ad9384dd88489654e380
+// flow-typed version: e9374c5fe9/moment_v2.3.x/flow_>=v0.25.x
```

Comment:
I got this by running `flow-typed install moment`. It looked at our dependency version and used the one that fit.